src/js/script.js

```diff
@@ -189,6 +189,7 @@
             }
           }
         }
+        thisProduct.dom.priceElem.innerHTML = price;
       }
       // update calculated price in the HTML
       thisProduct.priceElem.innerHTML = price;
```

Comment:
Tutaj powinnaś się odwołać do właściwości `dom` w obiekcie `thisProduct`
```suggestion
      thisProduct.dom.priceElem.innerHTML = price;
```